.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+# On every pull request, but only on push to master
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - '*'
```

Comment:
The `- '*'` line is unnecessary, you can just specify `tags:` and it will run on all tags.